unified-installer/astra-unified-installer.sh

```diff
@@ -690,7 +838,7 @@ join_rpath() {
     for (( i = 0; i < ${#args[@]}; i+=1 )); do
         args[i]="${args[i]#/}" # Remove starting slash if present
         args[i]="${args[i]%/}" # Remove trailing slash if present
-        if [ "$i" -eq 0 ]; then joined="${args[i]}"
+        if [ -z "$joined" ]; then joined="${args[i]}"
```

Comment:
Fixes an issue for when the first arg(s) are empty values. 

Given args:
```
join_rpath "" "astra-connector"
```
You'd get:
```
"/astra-connector"
```
Now you'll instead get:
```
"astra-connector"
```
The extra slash at the beginning was breaking some image tags depending on the values provided.